endpoints/openrtb2/sample-requests/invalid-whole/imp-ext-unknown-bidder.json

```diff
@@ -22,5 +22,5 @@
     }
   },
   "expectedReturnCode": 400,
-  "expectedErrorMessage": "Invalid request: request.imp[0].ext.prebid.bidder contains unknown bidder: noBidderShouldEverHaveThisName. Did you forget an alias in request.ext.prebid.aliases?\n"
+  "expectedErrorMessage": "Invalid request: request.imp[0].ext contains unknown bidder: 'noBidderShouldEverHaveThisName', ignoring"
```

Comment:
Interesting. Since the bidder name check is just a warning, this should fail because there are no bidders. Why is there not an error message with lack of bidders found?

Reply:
the test uses `assert.Contains()` checking that `expectedErrorMessage` is within `actualErrorMessage`. 
`actualErrorMessage` contained 2 error messages (this one was added as `errortypes.Warning`) and one of them was added as a proper error: `Invalid request: request.imp[0].ext.prebid.bidder must contain at least one bidder` - so request failed with this one.  this has been updated to obviously reflect the error message it fails with